grammars/erlang.bnf

```diff
@@ -321,6 +324,32 @@ argument_definition_list ::= '(' argument_definition? (',' argument_definition)*
 clause_guard ::= when <<guarded guard>> {pin=1}
 clause_body ::= '->' exprs {pin=1}
 
+app_expression ::= empty '{' 'application' ',' module_ref ',' app_parameters '}' {elementType=tuple_expression}
```

Comment:
Do not pin on empty, as empty is always matched. What if you, or somebody else adds another rule after your app_expression? It just won't match because you always pin your rule. Don't do that.